extensions/blocks/instagram-gallery/edit.js

```diff
@@ -218,6 +211,12 @@ const InstagramGalleryEdit = props => {
 						</PanelRow>
 					</PanelBody>
 					<PanelBody title={ __( 'Gallery Settings', 'jetpack' ) }>
+						<div className="wp-block-jetpack-instagram-gallery__count-notice">{ noticeUI }</div>
+						{ isLoadingGallery && (
+							<div className="wp-block-jetpack-instagram-gallery__gallery-loading">
+								<Spinner />
+							</div>
+						) }
```

Comment:
I'm not a fan of absolutely positioned elements!
What about inserting the spinner into the control label instead?

<img width="270" alt="Screenshot 2020-04-09 at 11 33 24" src="https://user-images.githubusercontent.com/2070010/78886582-5cbeb680-7a56-11ea-8309-4b4b459e5dc9.png">

Example code:
```jsx
<RangeControl label={
  <>
    Number of Posts
    <span className="wp-block-jetpack-instagram-gallery__gallery-loading">
      <Spinner />
    </span>
  </>
} />
```
```css
.wp-block-jetpack-instagram-gallery__gallery-loading .components-spinner {
	vertical-align: bottom;
}
```

Reply:
Yeh, I dislike absolute positioning as well, doesn't generally play nice with responsive layout - this was a quick hack for the sake of proving the concept, and I overlooked the fact that you could pass a component into the label prop - will remember this for the future, but it turned out we don't need this loader if we go with @scruffian's approach of just loading 30 up front.